src/frontend/src/lib/i18n/en.json

```diff
@@ -127,7 +127,8 @@
 			"erc20_user_tokens": "Error while loading the ERC20 user tokens.",
 			"erc20_user_token": "Error while enabling the ERC20 twin token.",
 			"icrc_custom_token": "Error while loading the ICRC custom token.",
-			"loading_wallet_timeout": "Your wallet requires some initial data which has not been loaded yet. Please try again."
+			"loading_wallet_timeout": "Your wallet requires some initial data which has not been loaded yet. Please try again.",
+			"allow_signing": "An issue occurred while setting up your wallet for signing. You've been signed out. Please sign in again, and if the issue persists, contact support."
```

Comment:
Theoretically, in the user view, he is not yet signed in: maybe we can remove the "You've been signed out."

Or not?

Reply:
Sure!